core/src/main/java/de/danoeh/antennapod/core/feed/FeedItem.java

```diff
@@ -388,6 +391,22 @@ public boolean hasChapters() {
         return hasChapters;
     }
 
+    public void setAutoDownload(boolean autoDownload) {
+        this.autoDownload = autoDownload;
+    }
+
+    public boolean getAutoDownload() {
+        return this.autoDownload;
+    }
+
+    public boolean isAutoDownloadable() {
+        return this.hasMedia() &&
+                false == this.getMedia().isPlaying() &&
```

Comment:
it's just that the "!" at the beginning is easy to miss. 
turning it around has some nice advantages, one is readability: "if it is false that media is playing..."

if(myvar = false) .... if you put false or true first, errors where you set a variable instead of comparing it cannot happen